tablo/api.py

```diff
@@ -201,16 +203,37 @@ def apply_edits(self, request, **kwargs):
 
         if original_time_extent:
             new_time_extent = feature_service_layer.get_raw_time_extent()
-            if (new_time_extent[0] != original_time_extent[0] or
-                new_time_extent[1] != original_time_extent[1]):
+            if (
+                new_time_extent[0] != original_time_extent[0] or
+                new_time_extent[1] != original_time_extent[1]
+            ):
```

Comment:
Can this be refactored so that the conditional is a one-liner?  

```
has_new_time_extent = (
    new_time_extent[0] != ...
)

if has_new_time_extent:
    response_obj['new_time_extent'] = json.dumps(new_time_extent)
```